VS_Project/Pokerapp/Pokerapp/delete-ajax.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@using WebMatrix.Data
+
+@{
+    //connection database
+    string connectionString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database.mdf;Integrated Security=True";
+    string provider = "System.Data.SqlClient";
+    Database db = Database.OpenConnectionString(connectionString, provider);
+
+    //get urlcode
+    //string url = Request.Url.AbsolutePath;
+    //string urlcode = url.Substring(url.Length - 7);
+
+    int room_code = 1234567;
```

Comment:
Dus... Wij hebben nu alleen ajax als je roomcode toevallig 1234567 is? Of is dit iets wat je er nog uit moet halen?
Heb je je code nagecheckt voordat je hem hebt gepusht? En heb je hem met meerdere verschillende roomcodes en settings getest?